### ISSUE TITLE
[roq-fix-bridge] Broadcast

### ISSUE THREAD

**Issue by thraneh** [roq-trading/roq-issues#167]
The Deribit index is published on a separate symbol than the instruments. There are use-cases where it makes more sense copy the index value into the instrument.

For example
* BTC-DERIBIT-INDEX is the index
* BTC-PERPETUAL is the instrument

Copying the `StatisticsUpdate`'s from the index to the futures can be done by adding something like this to the config

```
[broadcast]                                                                                                             
  [broadcast.index_to_all_futures]                                                                                      
  exchange="deribit"                                                                                                    
  source_regex="^(\\w+)-DERIBIT-INDEX$"                                                                                 
  targets_regex=[                                                                                                       
    "^(\\w+)-PERPETUAL$",                                                                                               
    "^(\\w+)-[\\d+]\\w+$"                                                                                               
  ]  
```

All regex are used to extract keys. Matching is done through these keys.

For example, the above regex examples would extract
* "BTC" from "BTC-DERIBIT-INDEX" (source)
* "BTC" from "BTC-PERPETUAL" (target)
* "BTC" from "BTC-22JAN22" (target)

Thus, for these symbols, we can establish the following broadcast:

"BTC-DERIBIT-INDEX" →{ "BTC-PERPETUAL", "BTC-22JAN22" }

In other words: 1 to many.

Constraints:
* A symbol is included in the broadcast (as either source or target) **ONLY IF** a key can be extracted.
* It is **NOT ALLOWED** to have more than 1 source symbol mapped to the same key
